demo_formats.py

```
@@ -0,0 +1,78 @@
+
+def main():
+    """Demo all three output formats."""
+    base_url = "http://example.com"
+    output_dir = Path("demo_output")
+    output_dir.mkdir(exist_ok=True)
+    
+    # Convert to all three formats
+    formats = {
+        "markdown": OutputFormat.MARKDOWN,
+        "json": OutputFormat.JSON,
+        "xml": OutputFormat.XML
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)